<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QMainWindow" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>655</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="attendance_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Your Files </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eventUpdate_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Attendance </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Updates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eventUpdates_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for Attatchments</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Announcements</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="announcement_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="announcment_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for Attatchments</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMail_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Category</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="all_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fe_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="se_checkbox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="te_checkbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="be_checkbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ce_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="it_checkbox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ppt_checkbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PPT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="extc_checkbox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EXTC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mech_checkbox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MECH</string>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>60</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>260</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>405</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionUpload_Student_Data"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionUpload_Student_Data">
   <property name="text">
    <string>Upload Student Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
